Sources/AWSLambdaRuntime/Lambda+Codable.swift

```diff
@@ -130,3 +131,17 @@ extension JSONEncoder: LambdaCodableEncoder {
         return buffer
     }
 }
+
+extension JSONEncoder {
+    /// Convenience method to allow encoding json directly into a String. It can be used to encode a payload into an APIGateway.V2.Response's body.
+    public func encodeAsString<T: Encodable>(_ value: T) throws -> String {
+        try String(decoding: self.encode(value), as: Unicode.UTF8.self)
+    }
+}
+
+extension JSONDecoder {
+    /// Convenience method to allow decoding json directly from a String. It can be used to decode a payload from an APIGateway.V2.Request's body.
+    public func decode<T: Decodable>(_ type: T.Type, from string: String) throws -> T {
```

Comment:
can be refactored as
public func decode<Element: Decodable>(_ type: Element.Type, from string: String) throws -> Element